Unity-Technologies/com.unity.multiplayer.samples.coop
# Fix: Removed pop-up issues during loading screen in Boss Room [MTT-3787] [MTT-5711]

### Description

I adjusted the sort order of different UI elements in order to display the RNSM window as well as the PopupPanel displaying connection attempts after a disconnect in front of the loading screen.

To verify the changes are correct:
1. Open Boss Room in the editor
2. Create a clone via parrel sync and open clone
3. Go into play mode in both editors, open RNSM window via pressing S
4. Create game session as host via direct IP in one editor and join the game with the other one
5. Reach post game (e.g. via pressing cheat option)
6. Watch that pop-ups appear in correct order during loading screens

### Issue Number(s)

[MTT-3787](https://jira.unity3d.com/browse/MTT-3787)
[MTT-5711](https://jira.unity3d.com/browse/MTT-5711)

### Contribution checklist
 - [ ] Tests have been added for boss room and/or utilities pack
 - [ ] Release notes have been added to the [project changelog](../CHANGELOG.md) file and/or [package changelog](../Packages/com.unity.multiplayer.samples.coop/CHANGELOG.md) file
 - [ ] An Index entry has been added in readme.md if applicable
